engine/packet.py

```diff
@@ -99,14 +105,14 @@ def __init__(self, phase, p_weight, acc, n_dist, rot, last_n, vel, next_n, coord
     def __str__(self):
         '''
         Returns: string of the following format using correcting size of args
-        "phase:0;p_weight:00.0;acc:0.00;n_dist:00.0;rot:00.00;last_n:000.00,000.00;vel:0.00;next_n:000.00,000.00;coord:000.00,000.00;batt:000;ctrl:1"
+        "phase:0;p_weight:00.0;acc:0.00,0.00,0.00;n_dist:00.0;rot:00.00;last_n:000.00,000.00;vel:0.00;next_n:000.00,000.00;coord:000.00,000.00;batt:000;ctrl:1"
         '''
         args = [("phase:", self.phase), (";p_weight:", fix_data_size(self.p_weight, 2, 1)),
-                (";acc:", (fix_data_size(self.acc[0], 1, 2)+","+fix_data_size(self.acc[1], 1, 2))+","+fix_data_size(self.acc[1], 1, 2)), 
+                (";acc:", fix_values_size(self.acc, 1, 2)), 
                 (";n_dist:", fix_data_size(self.n_dist, 2, 1)),
-                (";rot:", fix_data_size(self.rot, 2, 2)), (";last_n:", fix_tuple_size(self.last_n, 3, 2)),
-                (";vel:", fix_data_size(self.vel, 1, 2)), (";next_n:", fix_tuple_size(self.next_n, 3, 2)),
-                (";coord:", fix_tuple_size(self.coord, 3, 2)),
+                (";rot:", fix_data_size(self.rot, 2, 2)), (";last_n:", fix_values_size(self.last_n, 3, 2)),
+                (";vel:", fix_data_size(self.vel, 1, 2)), (";next_n:", fix_values_size(self.next_n, 3, 2)),
+                (";coord:", fix_values_size(self.coord, 3, 2)),
```

Comment:
fixing the string version of each value to match the specification of a packet string. For example, the x coord may be 0.0 but to match the specification of the packet string, it should be adjusted to "000.00".